js/preserveData.js

```diff
@@ -0,0 +1,21 @@
+const userName = document.querySelector('#userName');
+const userEmail = document.querySelector('#email');
+const userMessage = document.querySelector('#message');
+const submissionBtn = document.querySelector('#submit');
+
+window.addEventListener('load', () => {
+  const getUserData = JSON.parse(localStorage.getItem('setUserData'));
+  userName.value = getUserData.name;
+  userEmail.value = getUserData.email;
+  userMessage.value = getUserData.msg;
+});
+
+// set userDate to localStorage
+const setUserData = { name: '', email: '', msg: '' };
+
+submissionBtn.addEventListener('click', () => {
+  setUserData.name = userName.value;
+  setUserData.email = userEmail.value;
+  setUserData.msg = userMessage.value;
+  localStorage.setItem('setUserData', JSON.stringify(setUserData));
+});
```

Comment:
- [ ] Great job preserving the data in the browser :clap: :sunglasses: However data should be saved in the browser when the user changes the content of any input field. But you're saving the data when the user clicks on the button submit, Kindly check it